internal/pkg/testproject/project.go

```diff
@@ -132,9 +135,40 @@ func (p *Project) Clear() {
 		}
 	}
 
+	p.clearSchedules()
+
 	p.logf("Test project cleared | %s", time.Since(startTime))
 }
 
+// Clear deletes all schedules.
+func (p *Project) clearSchedules() {
+	logger, _ := utils.NewDebugLogger()
+	hostName, _ := p.api.GetSchedulerApiUrl()
+	schedulerApi := scheduler.NewSchedulerApi(
+		hostName,
+		p.api.Token().Token,
+		context.Background(),
+		logger,
+		true,
+	)
+
+	pool := schedulerApi.NewPool()
+
+	// Load schedules
+	schedules, err := schedulerApi.ListSchedules()
+	if err != nil {
+		assert.FailNow(p.t, fmt.Sprintf("cannot load schedules: %s", err))
+	}
+
+	// Delete all schedules
+	for _, schedule := range schedules {
```

Comment:
Vytvaranie `pool` by som dal nad tento `for`, skor nema zmysel, a bude to pokope.